src/ProjectNeon/Assets/Features/Effects/AllEffects.cs

```diff
@@ -8,7 +8,7 @@ public static class AllEffects
     {
         { EffectType.Nothing, e => new NoEffect() },
         { EffectType.HealFlat, e => new SimpleEffect(m => m.GainHp(e.IntAmount))},
-        { EffectType.PhysicalDamage, e => new SimpleEffect((src, m) => m.TakePhysicalDamage(src.State.Attack() * e.FloatAmount))},
+        { EffectType.PhysicalDamage, e => new PhysicalDamage(Convert.ToInt32(e.FloatAmount)) },
```

Comment:
`EffectData` already has `IntAmount` for using a float as an int. No explicit processing may happen outside of that.